spec/factories/pages.rb

```diff
@@ -0,0 +1,28 @@
+# == Schema Information
+#
+# Table name: pages
+#
+#  id           :integer          not null, primary key
+#  page_type    :integer
+#  title        :string
+#  content_html :string
+#  content      :string
+#  source_url   :string
+#  status       :integer
+#  screenshot   :string
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#  company_id   :integer
+#
+
+FactoryBot.define do
+  factory :page do
+    page_type { %i[bing angle.co linkedin crunchbase].sample }
```

Comment:
why only 1 has a TLD?

Reply:
it's OK, but move to constants in Page model

BING = 'bing'
...
Page::TYPES = [BING, ANGLECO, LINKEDIN....]

then
Page::TYPES.sample